uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
> Could we combine that here, and just factor out ALL config population to a separate utility script?

My understanding was that we were still far enough away from configdb that we need to keep thinking about how to manage our config flatfiles for the time being.  How close to you feel we are to actually being able to take on the configdb thing?